.github/workflows/ci.yml

```diff
@@ -30,6 +30,9 @@ jobs:
       - name: Build docs.rs
         run: cargo build --locked
 
+      - name: Run clippy
+        run: cargo clippy -- -D clippy::all -D unused-imports
```

Comment:
I'd just do `-D warnings`.

Reply:
Hmm, it looks like `-D warnings` doesn't work: 
```
$ cargo clippy -- -D warnings
warning: constant item is never used: `DEFAULT_ID`
   --> src/web/mod.rs:516:5
    |
516 |     const DEFAULT_ID: i32 = 0;
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: `#[warn(dead_code)]` on by default
```

Reply:
How strange, it worked after a `cargo clean`. It must have been introduced by `cargo` caching warnings after `check`.